modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/views/ChangeTrackingRenderView.js

```diff
@@ -1386,7 +1437,17 @@ export default function ChangeTrackingRenderView({
 								)}
 						</div>
 
-						<div className="entry-description">{description}</div>
+						<div className="entry-description">
+							<span>{description} </span>
+
+							<span>
+								{Liferay.Language.get('workflow-status')}:{' '}
```

Comment:
{} is not needed, but if you prefer using it, it should be used for both `: `.

Reply:
@pei-jung 
I'm having trouble figuring out how to add the feature flag here. I looked in other js files and implemented this:

```
<div className="entry-description">
	<span>{description} </span>

	{Liferay.FeatureFlags['LPD-10703'] ? (
		<span>
			{Liferay.Language.get('workflow-status')}

			{': '}
		</span>

		<WorkflowStatusLabel
			workflowStatus={workflowStatus}
		/>
	) : <span></span>}
</div>
```
But I'm getting this build error:
```
SyntaxError: Adjacent JSX elements must be wrapped in an enclosing tag. Did you want a JSX fragment <>...</>? (1450:9)
  1448 | 								</span>
  1449 | 
> 1450 | 								<WorkflowStatusLabel
       | 								^
  1451 | 									workflowStatus={workflowStatus}
  1452 | 								/>
  1453 | 							) : <span></span>}

```

Reply:
I figured it out.